src/ui/map.js

```diff
@@ -563,6 +563,7 @@ class Map extends Camera {
     /**
      * Returns the map's geographical bounds. When the bearing or pitch is non-zero, the visible region is not
      * an axis-aligned rectangle, and the result is the smallest bounds that encompasses the visible region.
+     * @returns {LngLatBounds} The geographical bounds of the map as {@link LngLatBounds}.
```

Comment:
There are **a lot** of JSDoc descriptions that are really just a description of the return value. It may make more sense to move these descriptions to the `@return` section and use the description for more general information.